scripts/cmake/Modules/LibAddPlugin.cmake

```
@@ -637,6 +637,8 @@ function (add_plugin PLUGIN_SHORT_NAME)
 
 	if (BUILD_SHARED)
 		add_library (${PLUGIN_NAME} MODULE ${ARG_SOURCES} ${ARG_OBJECT_SOURCES})
+		set_target_properties (${PLUGIN_NAME} PROPERTIES PREFIX "libelektra")
+		set_target_properties(${PLUGIN_NAME} PROPERTIES OUTPUT_NAME "-plugin-${PLUGIN_NAME}")
```

Probably better to add the `elektra` PREFIX here

```suggestion
		set_target_properties(${PLUGIN_NAME} PROPERTIES OUTPUT_NAME "elektra-plugin-${PLUGIN_NAME}")
```

AFAIK that should result in `libelektra-plugin-PLUGIN_NAME.so` on Linux and `elektra-plugin-PLUGIN_NAME.dll` on Windows.